arcsi/model/user.py

```diff
@@ -7,6 +7,8 @@
 class User(db.Model, UserMixin):
     __tablename__ = "users"
     id = db.Column(db.Integer, primary_key=True)
+    fs_uniquifier = db.Column(db.String(64), nullable=False)
+    fs_webauthn_user_handle = db.Column(db.String(64), unique=True, nullable=True)
```

Comment:
we don't want to do webauthn so that is not needed.

Reply:
It was mentioned in Flask-Security's [change log](https://flask-security.readthedocs.io/en/stable/changelog.html#db-migration) for the 5.0.0 version.
I was not sure if it is needed or not, but I thought it's easier to add in case we will need it in the future.